### PR TITLE
libenv: add Linux parsing of /proc/net/ data

### PR DESCRIPTION
**Pull request by tzz**
as requested in https://github.com/cfengine/core/pull/2245

### CODE REVIEW

File: reference/special-variables/sys.markdown [hunk 1]
```diff
@@ -262,6 +262,531 @@ whether this is qualified or unqualified with a domain name.
...
...wStartRetrans TCPTimeouts TCPLossProbes TCPLossProbeRecovery TCPRenoRecoveryFail TCPSackRecoveryFail TCPSchedulerFailed TCPRcvCollapsed TCPDSACKOldSent TCPDSACKOfoSent TCPDSACKRecv TCPDSACKOfoRecv TCPAbortOnData TCPAbortOnClose TCPAbortOnMemory TCPAbortOnTimeout TCPAbortOnLinger TCPAbortFailed TCPMemoryPressures TCPSACKDiscard TCPDSACKIgnoredOld TCPDSACKIgnoredNoUndo TCPSpuriousRTOs TCPMD5NotFound TCPMD5Unexpected TCPSackShifted TCPSackMerged TCPSackShiftFallback TCPBacklogDrop TCPMinTTLDrop TCPDeferAcceptDrop IPReversePathFilter TCPTimeWaitOverflow TCPReqQFullDoCookies TCPReqQFullDrop TCPRetransFail TCPRcvCoalesce TCPOFOQueue TCPOFODrop TCPOFOMerge TCPChallengeACK TCPSYNChallenge TCPFastOpenActive TCPFastOpenActiveFail TCPFastOpenPassive TCPFastOpenPassiveFail TCPFastOpenListenOverflow TCPFastOpenCookieReqd TCPSpuriousRtxHostQueues BusyPollRxPackets TCPAutoCorking TCPFromZeroWindowAdv TCPToZeroWindowAdv TCPWantZeroWindowAdv TCPSynRetrans TCPOrigDataSent TCPHystartTrainDetect TCPHystartTrainCwnd TCPHystartDelayDetect TCPHystartDelayCwnd TCPACKSkippedSynRecv TCPACKSkippedPAWS TCPACKSkippedSeq TCPACKSkippedFinWait2 TCPACKSkippedTimeWait TCPACKSkippedChallenge TCPWinProbe TCPKeepAlive
```
Comment (kacf) [inline reference/special-variables/sys.markdown]: @nickanderson: Do you know if a long line like this will render correctly when in a code block?

Comment (kacf) [inline reference/special-variables/sys.markdown]: I guess we'll take a chance!

Comment (nickanderson) [inline reference/special-variables/sys.markdown]: Yep: https://docs.cfengine.com/docs/master/reference-special-variables-sys.html#sys-inet

**Comment by tzz** [comment]
I know that was a lot of text but I wanted to show actual data you might see in that file. Thanks for verifying!

**Pull request merged**